### PR TITLE
add doc for information_schema.variables_info

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/35995

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by djshow832 — COMMENTED** [review]
Also update `information-schema/information-schema.md` and `TOC.md`.

File: information-schema/information-schema.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Information Schema 提供了一种查看系统元数据的 ANSI 标准方法。
 | `TABLE_PRIVILEGES`                                                                                                         | TiDB 未实现，返回零行。                                                           |
 | `TRIGGERS`                                                                                                                 | TiDB 未实现，返回零行。                                                           |
 | [`USER_PRIVILEGES`](/information-schema/information-schema-user-privileges.md)                                             | 汇总与当前用户相关的权限。                                                        |
+| [`VARIABLES_INFO`](/information-schema/information-schema-variables-info.md)                                               | 提供 TiDB 系统变量的信息                                                |
```
Comment (tangenta) [inline information-schema/information-schema.md:47]:
```suggestion
| [`VARIABLES_INFO`](/information-schema/information-schema-variables-info.md)                                               | 提供 TiDB 系统变量的信息。                                               |
```

File: information-schema/information-schema-variables-info.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: VARIABLES_INFO
+summary: 了解 information_schema 表 `VARIABLES_INFO`。
+---
+
+# VARIABLES_INFO
+
+`VARIABLES_INFO` 可用于查看当前 TiDB 集群/实例系统变量的默认值、当前值以及 Scope 等信息。
```
Comment (ran-huang) [inline information-schema/information-schema-variables-info.md:8]:
```suggestion
`VARIABLES_INFO` 可用于查看当前 TiDB 集群/实例系统变量的默认值、当前值以及作用域等信息。
```

File: information-schema/information-schema-variables-info.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| auto_increment_increment          | SESSION,GLOBAL  | 1             | 1             |         1 |     65535 | NULL            | NO      |
+| auto_increment_offset             | SESSION,GLOBAL  | 1             | 1             |         1 |     65535 | NULL            | NO      |
++-----------------------------------+-----------------+---------------+---------------+-----------+-----------+-----------------+---------+
+3 rows in set (0.02 sec)
+```
+
+`VARIABLES_INFO` 表中列的含义如下：
+
+* `VARIABLE_NAME`：系统变量名称。
+* `VARIABLES_SCOPE`：系统变量的应用范围。`SESSION` 表示当前 session 可见；`INSTANCE` 表示当前 TiDB 实例可见；`GLOBAL` 表示集群内可见。
```
Comment (ran-huang) [inline information-schema/information-schema-variables-info.md:53]:
```suggestion
* `VARIABLES_SCOPE`：系统变量的作用域。`SESSION` 表示当前 session 可见；`INSTANCE` 表示当前 TiDB 实例可见；`GLOBAL` 表示集群内可见。
```

File: information-schema/information-schema-variables-info.md (hunk 2 above)
Comment (ran-huang) [reply information-schema/information-schema-variables-info.md:8]: 与[系统变量](https://docs.pingcap.com/zh/tidb/stable/system-variables)里保持一致

**Review by ran-huang — APPROVED** [review]
Rest LGTM. Nice doc!

**Review by djshow832 — APPROVED** [review]

File: information-schema/information-schema-variables-info.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+USE information_schema;
+DESC variables_info;
+```
+
+```sql
++-----------------+---------------------+------+------+---------+-------+
+| Field           | Type                | Null | Key  | Default | Extra |
++-----------------+---------------------+------+------+---------+-------+
+| VARIABLE_NAME   | varchar(64)         | NO   |      | NULL    |       |
+| VARIABLES_SCOPE | varchar(64)         | NO   |      | NULL    |       |
```
Comment (tangenta) [inline information-schema/information-schema-variables-info.md:22]:
```suggestion
| VARIABLE_SCOPE  | varchar(64)         | NO   |      | NULL    |       |
```

File: information-schema/information-schema-variables-info.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+title: VARIABLES_INFO
+summary: 了解 information_schema 表 `VARIABLES_INFO`。
+---
+
+# VARIABLES_INFO
+
+`VARIABLES_INFO` 可用于查看当前 TiDB 集群/实例系统变量的默认值、当前值以及作用域等信息。
```
Comment (shichun-0415) [inline information-schema/information-schema-variables-info.md:8]: It is not recommended to use / to express the meaning of "or".
```suggestion
`VARIABLES_INFO` 可用于查看当前 TiDB 集群或实例的系统变量默认值、当前值以及作用域等信息。
```

File: information-schema/information-schema-variables-info.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+{{< copyable "sql" >}}
```
Comment (shichun-0415) [inline information-schema/information-schema-variables-info.md:10]: We no longer use this code snippet. FYI: [Docs Maintenance Changelog](https://pingcap.feishu.cn/wiki/wikcnpK4JnRsc9Zje25DLDWidLb)
```suggestion
```

File: information-schema/information-schema-variables-info.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+| MIN_VALUE       | bigint(64)          | YES  |      | NULL    |       |
+| MAX_VALUE       | bigint(64) unsigned | YES  |      | NULL    |       |
+| POSSIBLE_VALUES | varchar(256)        | YES  |      | NULL    |       |
+| IS_NOOP         | varchar(64)         | NO   |      | NULL    |       |
++-----------------+---------------------+------+------+---------+-------+
+8 rows in set (0.00 sec)
+```
+
+{{< copyable "sql" >}}
+
```
Comment (shichun-0415) [inline information-schema/information-schema-variables-info.md:34]:
```suggestion
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by CbcWestwolf — APPROVED** [review]

**Pull request merged**